sopel/logger.py

```diff
@@ -33,10 +33,15 @@ def formatException(self, exc_info):
 
 
 def setup_logging(bot):
-    level = bot.config.core.logging_level or 'WARNING'
-    logging.basicConfig(level=level)
+    cfg = bot.config.core
```

Comment:
I'd rather not shortcut-reference the core config section like this. It makes the code harder to understand when coming back to it months/years later.

Reply:
Agree on that!

Reply:
corrected in latest version in git